# This is a message to hold the timing statistics of the obca controllers
Header header

float64 tv_obs_gen
float64 strategy_pred
float64 ref_collision_check
float64 hyperplane_gen
float64 warm_start_solve
float64 obca_solve
float64 safety_solve
float64 ebrake_solve
float64 total
